pkgs/development/libraries/wlroots/default.nix

```diff
@@ -127,9 +127,11 @@ rec {
   wlroots_0_17 = generic {
     version = "0.17.2";
     hash = "sha256-Of9qykyVnBURc5A2pvCMm7sLbnuuG7OPWLxodQLN2Xg=";
+    extraNativeBuildInputs = [
+      hwdata
```

Comment:
Why move hwdata to nativeBuildInputs?

Reply:
There are some related discussions: https://lists.openembedded.org/g/openembedded-devel/topic/94909525